terraform/docdb/main.tf

```diff
@@ -0,0 +1,97 @@
+terraform {
+  required_version = "~> 1.0"
+
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 3.27"
```

Comment:
Latest is `4.48`

Reply:
Updated

Reply:
After change, the `ci_terraform` workflow fails

Reply:
The issue was something else. We don't always remove the Grafana API keys (like, when the CI fails) and expired keys still count in AWS quotas.
I cleaned up in Grafana and it should work now.
For ref: https://docs.aws.amazon.com/grafana/latest/userguide/AMG_quotas.html